api/controllers/UserController.ts

```diff
@@ -126,4 +127,10 @@ export class UserController {
     await this.userSocialMediaService.deleteSocialMediaByUuid(user, params.uuid);
     return { error: null };
   }
+
+  @Patch('/canSeeAttendance')
+  async changeCanSeeAttendance(@AuthenticatedUser() user: UserModel): Promise<ChangeCanSeeAttendanceResponse> {
+    await this.userAccountService.changeCanSeeAttendance(user);
+    return { error: null, canSeeAttendance: user.canSeeAttendance };
+  }
 }
```

Comment:
I don't think we necessarily need a separate route for this update. `patchCurrentUser` is meant to handle all updates to the UserModel, so we can have this update happen there.